test/redshift_test.rb

```diff
@@ -0,0 +1,35 @@
+require_relative "test_helper"
+
+class TestRedshift < Minitest::Test
+  include TestGroupdate
+  include TestDatabase
+
+  def setup
+    super
+    @@setup ||= begin
+      abort("REDSHIFT_URL environment variable must be set in order to run tests") unless ENV["REDSHIFT_URL"].present?
```

Comment:
@ankane had to resort to abort to get Minitest not to run the tests anyway, but it's much cleaner now